### DISCUSSION TITLE
Delete & Recreate Schedule with same Id don't works

### DISCUSSION THREAD

**Discussion by kapilbablani** [jobrunr/jobrunr#457] [Q&A]
Hello Everyone, with JobRunr in my project I am trying to create the schedule and it runs well on reaching the time. But when I delete the  schedule by calling the API to delete the schedule on Id and recreate the job with same Id again with same different time, the schedule don't run at the newly scheduled time.
Any known issue or bug is it ?
Spring Boot Version- 2.6.7
JobRunR Version- 5.0.1

Any help would be appreciated !

**Comment by rdehuyss** [comment]
This is a regression in in JobRunr 5 and I'm looking into a good solution. Please roll back to JobRunr 4 for now while I think of a solution.

**Reply by kapilbablani** [reply]
@rdehuyss Any specific version of JobRunr 4 ?

**Reply by rdehuyss** [reply]
4.0.10

**Reply by kapilbablani** [reply]
@rdehuyss Is the issue been fixed with version 5.x.x ??

**Reply by rdehuyss** [reply]
Fixed yes, released not yet. I hope to do this in coming days or early next week.

**Reply by kapilbablani** [reply]
@rdehuyss  Please let me know once the fix is released.

**Reply by rdehuyss** [reply]
Everything can be found in the release notes here in GitHub. So yes, it is released in 5.1.3.

Please star the repo if you're happy with JobRunr.
